aws/iox/alb.hcl

```diff
@@ -0,0 +1,474 @@
+ingester aws_alb module {
```

Comment:
Is this suppose to be in this repo? this is an IOX transform module. I think this is an internal implementation. This should not be exposed to the users of OME